### PR TITLE
Docsupdate

### PR DESCRIPTION
**Pull request by czurnieden**
Added some of the documentation @karel-m 's little perl script found in #135 . Please check for typeos etc. first.

### CODE REVIEW

File: tommath_class.h [hunk 1]
```diff
@@ -1133,12 +1201,8 @@
 #   define LTM_LAST
 #endif
 
-#include "tommath_superclass.h"
-#include "tommath_class.h"
```
Comment (karel-m) [inline tommath_class.h]: here you are reverting changes that were agreed in #127

File: tommath_class.h [hunk 2]
```diff
@@ -1133,12 +1201,8 @@
...
+#include <tommath_superclass.h>
+#include <tommath_class.h>
 #else
 #   define LTM_LAST
 #endif
-
-/* ref:         $Format:%D$ */
-/* git commit:  $Format:%H$ */
-/* commit time: $Format:%ai$ */
```
Comment (karel-m) [inline tommath_class.h]: missing final set of comments causes `./helper.pl -a` to fail

File: tommath_class.h [hunk 3]
```diff
@@ -1,15 +1,3 @@
...
- * LibTomMath is a library that provides multiple-precision
- * integer arithmetic as well as number theoretic functionality.
- *
- * The library was designed directly after the MPI library by
- * Michael Fromberger but has been written from scratch with
- * additional optimizations in place.
- *
- * The library is free for all purposes without any express
- * guarantee it works.
- */
```
Comment (karel-m) [inline tommath_class.h]: missing "standard" initial comment causes `./helper.pl -a` to fail

Comment (czurnieden) [reply tommath_class.h]: Sorry, @karel-m my bad, had the wrong base initially. Should be the correct one now.

**Pull request merged**